### DISCUSSION TITLE
Removing hover:underline from child elements

### DISCUSSION THREAD

**Discussion by eduardonwa** [tailwindlabs/tailwindcss#3836] [Help] [answered]
I have something like this:

```
<p class="bg-yellow-500 p-2">
    Assign category or, 
    <a 
        href="#"
        class="transition ease-in-out hover:underline hover:text-white"
    >
        create a new one
        <span style="font-size:20px; padding-left: 5px;">
            &#x1F913; // nerdy emoji
        </span>
    </a>
</p>
```
I am currently applying underline when I hover over a text, and I wish to remove the underline effect from one of the child element; the nerdy emoji. 

https://tailwindcss.com/docs/text-decoration#hover 
According to that, if I wanted to remove it would be just a matter of ```no-underline``` but what it doesn't say and im not sure how to do that, is how to remove it from a child element.

**Comment by dance2die** [comment, marked as answer]
According to https://stackoverflow.com/a/31297407/4035, `text-decoration` propagates to "inline" descendants.
So you can stop the decoration propagation by setting `span` as `inline-block` does the job.

If you have a control over your code, you might want to span around the text, and style underline on it only, though.

Here is the working Demo: https://play.tailwindcss.com/Rr8qs4Cmql

![text decoration](https://user-images.githubusercontent.com/8465237/112427955-d8218280-8d10-11eb-81fc-dbd4833df50d.gif)

**Reply by eduardonwa** [reply]
Thanks!

**Reply by dance2die** [reply]
yw and have fun~
